projects/online/online/main.py

```diff
@@ -460,8 +476,35 @@ def main(
         server,
         outdir,
     )
-    pastro_process = Process(target=pastro_subprocess, args=args, daemon=True)
+    pastro_process = Process(
+        target=pastro_subprocess,
+        args=args,
+    )
     pastro_process.start()
+    subprocesses.append(pastro_subprocess)
+
+    # Define a function to clean up all
+    # of these processes if we exit the
+    # main process
+    def cleanup_subprocesses():
```

Comment:
Lets move this into the `subprocesses` module in the same file as the `subprocess_wrapper`, maybe call it `utils.py`